pkgs/top-level/coq-packages.nix

```diff
@@ -70,6 +70,7 @@ let
       pocklington = callPackage ../development/coq-modules/pocklington {};
       QuickChick = callPackage ../development/coq-modules/QuickChick {};
       relation-algebra = callPackage ../development/coq-modules/relation-algebra {};
+      reglang = callPackage ../development/coq-modules/reglang {};
```

Comment:
Please try to list the packages in alphabetical order…

Reply:
Oops, my bad.